internal/indexmanagement/scripts.go

```diff
@@ -254,27 +293,14 @@ exit 1
 const deleteScript = `
 set -uo pipefail
 ERRORS=/tmp/errors.txt
-CONNECT_TIMEOUT=${CONNECT_TIMEOUT:-30}
-echo "" > $ERRORS
 
-echo "Index management delete process starting"
+source /tmp/scripts/indexManagement
 
-writeIndices=$(curl -s $ES_SERVICE/_alias/${POLICY_MAPPING}-write \
-  --cacert /etc/indexmanagement/keys/admin-ca \
-  --connect-timeout ${CONNECT_TIMEOUT} \
-  -H"Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" \
-  -HContent-Type:application/json \
-  --retry 5 \
-  --retry-delay 5)
-
-if [ -z "$writeIndices" ]; then
-  echo "Received an empty response from elasticsearch -- server may not be ready"
-  exit 1
-fi
+echo "" > $ERRORS
 
-jsonResponse="$(echo [$writeIndices] | sed 's/}{/},{/g')"
+echo "Index management delete process starting"
 
-if ! writeIndex="$(python /tmp/scripts/getWriteIndex.py "${POLICY_MAPPING}-write" "$jsonResponse")" ; then
+if ! writeIndex="$(getWriteIndex "${POLICY_MAPPING}-write")" ; then
```

Comment:
Can we spell out the `echo $writeIndex` as a proper exit message to understand what failed where?

Reply:
The `getWriteIndex` function should do that, no?